tool/tbot/config/config_storage.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package config
+
+import "github.com/gravitational/trace"
+
+const STORAGE_DEFAULT_PATH = "/var/lib/teleport/bot"
```

Comment:
How about something like this?

```suggestion
const DefaultStoragePath = filepath.Join(defaults.DataDir, "bot")
```

You'll probaby have to drop the const.

Reply:
I like that a lot, and I'll unexport it to avoid any mutation (can add an accessor function if needed later)